### ISSUE TITLE
Navigate to URL

### ISSUE THREAD

**Issue by jpmanson** [widgetti/solara#456]
I didn't find any way to navigate an external URL directly from the app. Something like for example:
`solara.use_router().push('https://www.google.com')`

Although you can create HTML buttons or links to go to other pages, they require a click from the user.

It would be very nice if this "navigate to URL" possibility could be added.

Thanks

**Comment by iisakkirotko** [comment]
Hi @jpmanson! 

Indeed the solara router does not support this, but you could always do this (for instance via a custom `component_vue`) with javascript. Running `window.location.href = 'https://solara.dev';` should do it.

**Comment by jpmanson** [comment]
Thanks @iisakkirotko ! I will try it

**Comment by maartenbreddels** [comment]
With 11c6094f3c47528d40f8be951c1d322e454f98ea the following should work (should be released in version >1.28.0)

```python
import solara


@solara.component
def Page():
    router = solara.use_router()

    def my_click_handler():
        router.push('https://www.google.com/')

    solara.Button("go", on_click=my_click_handler)
```
